nginx-controller/nginx/configurator.go

```diff
@@ -68,6 +68,70 @@ func (cnf *Configurator) addOrUpdateIngress(ingEx *IngressEx) {
 	cnf.ingresses[name] = ingEx
 }
 
+// AddOrUpdateIngress adds or updates NGINX configuration for the Ingress resources with Mergeable Types
+func (cnf *Configurator) AddOrUpdateMergableIngress(mergeableIngs MergeableIngresses) error {
+	cnf.addOrUpdateMergableIngress(mergeableIngs)
+
+	if err := cnf.nginx.Reload(); err != nil {
+		return fmt.Errorf("Error when adding or updating ingress %v/%v: %v", mergeableIngs.Master.Ingress.Namespace, mergeableIngs.Master.Ingress.Name, err)
+	}
+	return nil
+}
+
+func (cnf *Configurator) addOrUpdateMergableIngress(mergeableIngs MergeableIngresses) {
+	nginxCfg := IngressNginxConfig{}
```

Comment:
no need to declare IngressNginxConfig here

Reply:
Done